lib/cli.go

```diff
@@ -171,7 +171,11 @@ func CLI(program, version string, flags Flags) error {
 			Name:  "no-server",
 			Usage: "do not start embedded etcd server (do not promote from client to peer)",
 		},
-
+		cli.BoolFlag{
+			Name:   "no-network-standalone",
```

Comment:
Seems redundant, would `no-network` by itself be sufficient?
Can you elaborate more on the use case for this? Is it just standalone single host usage?

Reply:
Use case is indeed to have mgmt run (as root) on a single host without any TCP connections on localhost so an unprivileged user is not able to access the mgmt API and elevate to root.

I believe we discussed the name back then on irc and ended up with this. But `no-network` would be fine for me as well.